engine/src/main/java/org/terasology/rendering/particles/components/OverheadParticleSystem.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.particles.components;
+
+import org.terasology.entitySystem.Component;
+
+/**
+* The emitters of systems with this component will constantly float above the camera (for rain effects)
+* */
+public class OverheadParticleSystem implements Component {
```

Comment:
Hmm, potential confusion incoming. There might be `ParticleSystemComponent`s and `ParticleSystemSystem`s. I'd stick with the naming convention to end all components with `Component` and all systems with `System`, otherwise one would think that this is a system instead of a component.

Edit: the pattern is applied to the `ParticleSystemComponent` class already, so this is a call for consistency.